### PR TITLE
V1.3.3 new bt connection

### PR DESCRIPTION
**Pull request by brentpicasso**
Removed background service, added BT connection as local java bridge

### CODE REVIEW

File: autosportlabs/comms/androidcomms.py [hunk 1]
```diff
@@ -29,120 +15,48 @@ class PortNotOpenException(Exception):
...
     
     def isOpen(self):
-        return self._is_open
+        return self._bt_conn.isOpen()
     
     def open(self):
         print('Opening connection ' + str(self.port))
-        self.start_connection_process()
+        self._bt_conn.open(self.port)
+        print('after open!!!!!!!!!!!!!!!!')
```
Comment (ryandoherty) [inline autosportlabs/comms/androidcomms.py]: Only one exclamation point max per PEP8 standards.

Comment (brentpicasso) [inline autosportlabs/comms/androidcomms.py]: I was so excited

File: autosportlabs/racecapture/api/rcpapi.py [hunk 2]
```diff
@@ -23,6 +23,7 @@
 
 COMMS_KEEP_ALIVE_TIMEOUT        = 2
 
+NO_DATA_AVAILABLE_DELAY         = 0.1
```
Comment (ryandoherty) [inline autosportlabs/racecapture/api/rcpapi.py]: Could we get a comment above this describing what it is? I can't tell from the name.

File: java/src/com/autosportlabs/racecapture/BluetoothConnection.java [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+import java.util.UUID;
+import java.util.Set;
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.lang.reflect.Method;
+
+
+public class BluetoothConnection {
```
Comment (ryandoherty) [inline java/src/com/autosportlabs/racecapture/BluetoothConnection.java]: Can we get a short description of why this class exists and what it does? An entire class/file with no comments makes me worry about future devs opening this. Also some comments for any particularly hairy or unique sections would help.

**Comment by ryandoherty** [comment]
\o/ less code!

**Pull request merged**